### DISCUSSION TITLE
[Hubs Cloud AWS] Thumbnails no longer loading in the scene/avatar selection dialog

### DISCUSSION THREAD

**Discussion by esperecyan** [mozilla/hubs#4882] [Hubs Cloud Questions] [answered]
**Description**
Thumbnails no longer loading in the scene/avatar selection dialog.
Furthermore, creating a new avatar will not load the thumbnail.

**To Reproduce**
1. Create new AWS Cloudformation stack
1. Open the Select Avatar dialog on a room page
1. Upload avatar GLB file
1. Confirm that the thumbnails are no longer loaded

**Expected behavior**
Thumbnails will be loaded in the scene/avatar selection dialog.

**Screenshots**
![](https://user-images.githubusercontent.com/3625160/142365629-ea8dd5d9-e78f-4a81-be53-5ea3fa325615.png)

**Hardware** 
 - Hubs Cloud AWS 1.1.2
 - Device: Desktop
 - OS: Windows 10
 - Browser: Firefox 94.0.1

**Additional context**
The thumbnail URLs return 502 Bad Gateway.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1160)

**Comment by esperecyan** [comment, marked as answer]
I was able to work around this problem by following the steps in the comments below.
https://github.com/mozilla/hubs/discussions/4831#discussioncomment-1694508

**Comment by matthewbcool** [comment]
moving to hubs cloud discussion and linking this to https://github.com/mozilla/hubs/discussions/4831
